NSS-Day-Cohort-53/tabloid-full-stack-amicably-profound-insects
# Bw post details

#52
#### Changes Made
1. Made Changes to `PostList.js, PostDetails.js` directory.
1. Modified file `PostDetals.js` to include `show the title, content, author display name and publishDate of any selected post.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bw-PostDetails
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > ****Click on the Posts link in the navigation bar. It should yield a list of all published posts. There is a link on the right of the screen labeled "Details". Click that and you should see the Title, Author display name, Content, and Publish date.

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
